automation_tools/__init__.py

```diff
@@ -418,6 +418,11 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_scap_client():
```

Comment:
A task is required in order to use the `execute` on `product_install`. `execute` allows specify the host where the commands should be run and we need this to run the command on a machine or on a VM (if creating one).